src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -389,4 +389,10 @@ func Setup(
 			}}
 		}),
 	).Methods("GET", "OPTIONS")
+
+	r0mux.Handle("/account/whoami",
+		common.MakeExternalAPI("whoami", func(req *http.Request) util.JSONResponse {
+			return Whoami(req, deviceDB)
+		}),
+	).Methods("GET")
```

Comment:
I'm not sure what Dendrite's policy is, but considering a bunch of the GET routes above have OPTIONS, might as well do it here too?

Reply:
Yes

Reply:
Yeah not entirely sure what we're using that for at the moment, but best to keep it consistent.